src/index.html

```diff
@@ -7,8 +7,39 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Lumanosimo&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav-menu">
+          <li><a class="is-active nav__list" href="index.html">Apple</a></li>
```

Comment:
Add empty lines between multiline sibling blocks of HTML
<img width="976" alt="Screenshot 2023-10-13 at 13 39 31" src="https://github.com/mate-academy/layout_moyo-header/assets/64746943/441c4b71-69fe-4288-9847-4b6d3c2dd42d">